jenkins/release-workflows/release-chores.jenkinsfile

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@8.2.0', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+            timeout(time: 1, unit: 'HOURS')
+    }
+    agent {
+        docker {
+            label 'Jenkins-Agent-AL2023-X64-M54xlarge-Docker-Host'
+            image 'opensearchstaging/ci-runner:ci-runner-centos7-opensearch-build-v3'
+            registryUrl 'https://public.ecr.aws/'
+            alwaysPull true
+        }
+    }
+    parameters {
+        string(
+            name: 'RELEASE_VERSION',
+            description: "Required: On-going release version, e.g. 2.0.0",
+            trim: true
+        )
+        string(
+            name: 'RELEASE_CHORE',
+            description: "Required: What release chore you wanna do? e.g: 'add_rc_details_comment'",
```

Comment:
Where can we find these entries? Is a choice parameter better here?

Reply:
Thought of adding the choice parameter but I believe executing multiple tasks at once makes sense. I'll look into multi-select parameter option in the future. https://plugins.jenkins.io/multiselect-parameter/
For now you can find the options in the parameter description.